scripts/cbmc_json_parser.py

```diff
@@ -469,6 +474,13 @@ def construct_property_message(properties):
 
     return output_message
 
+def count_failures(properties):
```

Comment:
Don't we have this counter logic somewhere else where we print the execution summary? We should keep only one source of truth to the number of failures.

Reply:
Deleted this function, and returned the number of failures from the functions that construct the summary message.